Tests/SwiftGenKitTests/StoryboardsMacOSTests.swift

```diff
@@ -64,4 +64,18 @@ class StoryboardsMacOSTests: XCTestCase {
 
     XCTDiffContexts(result, expected)
   }
+
+  func testAllStoryboardsWithCustomName() {
+    let parser = StoryboardParser()
+    do {
+      try parser.parseDirectory(at: Fixtures.directory(sub: .storyboardsMacOS))
+    } catch {
+      print("Error: \(error.localizedDescription)")
+    }
+
+    let result = parser.stencilContext(sceneEnumName: "XCTStoryboardsScene", segueEnumName: "XCTStoryboardsSegue")
```

Comment:
For template stuff, such as `enumName`, these will be stored in `params` and don't affect SwiftGenKit. Should we have flags that change the generator's behaviour, I think we should have those as actual CLI documented flags/options.